packages/now-next/src/index.ts

```diff
@@ -1,20 +1,3 @@
-import buildUtils from './build-utils';
-import url from 'url';
-const {
-  createLambda,
-  debug,
-  download,
-  getLambdaOptionsFromFunction,
-  getNodeVersion,
-  getSpawnOptions,
-  getScriptName,
-  glob,
-  runNpmInstall,
-  runPackageJsonScript,
-  execCommand,
-  getNodeBinPath,
-} = buildUtils;
```

Comment:
This is necessary for legacy support

Reply:
Could you explain more? I just moved the code down.

Reply:
:oh:

Lets keep the `import buildUtils from './build-utils';` just above the deconstruction there.

Reply:
This change was done by the auto import formatter, I don't think we should be trying to fight against it.